electron-utils/menu-bar.ts

```diff
@@ -0,0 +1,47 @@
+import { app } from 'electron';
+
+// Edited version of a template menu-bar provided by the electron API,
+// refer to https://electronjs.org/docs/api/menu for more information.
+export const mainMenuTemplate: Electron.MenuItemConstructorOptions[] = [
+    {
+      label: 'File',
+      submenu: [
+        {
+          label: 'Quit CATcher', accelerator: 'CmdOrCtrl+Q', click() { app.quit(); }
+        }
+      ]
+    },
+    {
+      label: 'Edit',
```

Comment:
Let's breakup the menu definition further, with each submenu defined separately.
The exported object can then be defined like this:
`export const mainMenuTemplate: Electron.MenuItemConstructorOptions[] = [fileMenu, editMenu, viewMenu]`

The reader will be able to quickly see the menu's composition without reading the whole file.